src/target/riscv/riscv-013.c

```diff
@@ -4881,8 +4985,10 @@ static target_addr_t write_memory_progbuf_fill_batch(struct riscv_batch *batch,
 		log_memory_access64(address, value, size, /*is_read*/ false);
 		if (writes_per_element == 2)
 			riscv_batch_add_dm_write(batch, DM_DATA1,
-					(uint32_t)(value >> 32), false);
-		riscv_batch_add_dm_write(batch, DM_DATA0, (uint32_t)value, false);
+					(uint32_t)(value >> 32), false,
+					RISCV_DELAY_ABSTRACT_COMMAND);
```

Comment:
The write to DATA1 does not trigger the abstract command, so I believe `RISCV_DELAY_BASE` should be used here:

```suggestion
					(uint32_t)(value >> 32), false,
					RISCV_DELAY_BASE);
```

Reply:
Addressed.